api/dtos/responses/auth/auth_response.go

```diff
@@ -1,6 +1,6 @@
 package responses
 
 type AuthResponse struct {
-	AccessToken  string `json:"accessToken"`
-	RefreshToken string `json:"refreshToken"`
+	AccessToken  string `json:"access_token"`
+	RefreshToken string `json:"refresh_token"`
```

Comment:
Gebruik camelCasing. Dat is de naming convention voor JSON. Ik weet dat dit in de Swagger stond maar dan heeft Daan dat niet aangepast.

Reply:
Klopt, probleem is tho dat keycloak gebruik maakt van snake_case (zie https://www.keycloak.org/securing-apps/token-exchange onder Responses from a token exchange request). Vind het daarom beetje onnodig om nog een complete mapper te maken voor het mappen van de response van keycloak naar een auth_response, maakt ditte een hoop simpeler qua design en qua uitbreiding in de toekomst.

Reply:
Ah okay dat wist ik niet. Laat dan maar lekker zo 👍
Maar nu ik erover nadenk, als de access en refresh in de cookies gaan staan dan wil je dit toch niet als auth response? Hoe zie je dit in? Of wil je dat de API gateway de cookie mapping gaat doen? Lijkt me misschien alleen maar meer management voor de API Gateway.

Reply:
goed punt idd! we hebben er recentelijk over gepraat dus niet over na gedacht, fjixed it now naar token pair entity ipv